olm-catalog/serverless-operator/serverless-operator.package.yaml

```diff
@@ -2,3 +2,5 @@ packageName: serverless-operator
 channels:
 - name: techpreview
   currentCSV: serverless-operator.v1.5.0
+- name: 4.3
```

Comment:
Do we need a 4.4 channel too?

Reply:
Yes. Based on the Openshift Serverless Release Component Matrix, it looks like we should have a 4.4. channel as well. I'll go ahead and add it.